### PR TITLE
feat: Adding op-nethermind

### PR DESCRIPTION
**Pull request by parithosh**
The op-nethermind image is found here: `nethermindeth/nethermind:op-9b835cf`, it isn't stable/out yet so this is probably best left as a WIP for now.

The major missing piece: op-nethermind needs a config.json file, but op-node doesn't seem to output it. So we gotta find a way to template or generate that file for the network before this PR can have a chance to work.

### CODE REVIEW

**Comment by barnabasbusa** [comment]
Got an example of what kinda config file we are looking for?

**Comment by parithosh** [comment]
https://github.com/NethermindEth/nethermind/blob/949589fcdc6aa7ef15be410a455c74d4dcd32142/src/Nethermind/Nethermind.Runner/configs/op-sepolia.cfg#L40

Flcl will write a tiny thing to convert the genesis.json into the format they need

**Comment by barnabasbusa** [comment]
would be a prime time to just make nethermind read genesis config lol

File: static_files/chainspec_template/gen2spec.jq [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+    # Cancun
+    "eip4844TransitionTimestamp": .config.cancunTime|to_hex,
+    "eip4788TransitionTimestamp": .config.cancunTime|to_hex,
+    "eip1153TransitionTimestamp": .config.cancunTime|to_hex,
+    "eip5656TransitionTimestamp": .config.cancunTime|to_hex,
+    "eip6780TransitionTimestamp": .config.cancunTime|to_hex,
+    
+    #Prague
+    "rip7212TransitionTimestamp": .config.pragueTime|to_hex,
```
Comment (barnabasbusa) [inline static_files/chainspec_template/gen2spec.jq:133]: is this a typo? 

Isn't this supposed to be eip?

**Review by barnabasbusa — APPROVED** [review]

File: .github/tests/op-nethermind.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+optimism_package:
+  participants:
+    - el_type: op-nethermind
+      cl_type: op-node
```
Comment (barnabasbusa) [inline .github/tests/op-nethermind.yaml:4]:
```suggestion
    - el_type: op-geth
    - el_type: op-nethermind
      cl_type: op-node
```

**Pull request merged**